pennylane/resource/resource.py

```diff
@@ -14,6 +14,9 @@
 """
 Stores classes and logic to aggregate all the resource information from a quantum workflow.
 """
+from __future__ import annotations
```

Comment:
```suggestion
```

Reply:
This is needed for the type hints to work. Is there a problem with this import?